cleanupJavadoc.sh

```diff
@@ -1,2 +0,0 @@
-find release/doc -name "*.html" -exec sed -i "s/.*Generated by javadoc.*//" {} \;
```

Comment:
Why did you remove this file? I added it to cleanup the javadoc so that generating the JavaDoc doesn't result is a file change if the corresponding Java file did not change.